code/player/camera/SpectateCamera.cs

```diff
@@ -71,6 +71,21 @@ private Vector3 GetViewOffset()
 
             return player.EyeRot.Forward * -150 + Vector3.Up * 10;
         }
+
+        private List<TTTPlayer> GetAlivePlayers()
+        {
+            List<TTTPlayer> players = new List<TTTPlayer>();
+
+            foreach (Client client in Client.All)
+            {
+                if (client.Pawn is TTTPlayer player && player.LifeState == LifeState.Alive)
+                {
+                    players.Add(player);
+                }
+            }
+
+            return players;
+        }
```

Comment:
Could you put it into Player.cs?
Could you modify <https://github.com/TTTReborn/ttt-reborn/blob/main/code/player/Player.cs#L36-L49> as well to use `Client.All`?
Does it work for client and server? (so Clients know who is alive)?

Reply:
> Does it work for client and server? (so Clients know who is alive)?

Just tested, indeed, it is... That's bad :D

Reply:
Gonna add this method in the scoreboard PR

Reply:
Are we fine with my changes? I think it'll be better just to start spinning off any util methods we need.